RecoMuon/DetLayers/src/MuonGEMDetLayerGeometryBuilder.cc

```diff
@@ -23,10 +24,26 @@ MuonGEMDetLayerGeometryBuilder::~MuonGEMDetLayerGeometryBuilder() {}
 // Builds etaPartitions (for rechits)
 pair<vector<DetLayer*>, vector<DetLayer*> > MuonGEMDetLayerGeometryBuilder::buildEndcapLayers(const GEMGeometry& geo) {
   vector<DetLayer*> result[2];
-
   for (int endcap = -1; endcap <= 1; endcap += 2) {
     int iendcap = (endcap == 1) ? 0 : 1;  // +1: forward, -1: backward
 
+    // GE0
+    int station0 = 0;
+    for (int layer0 = GEMDetId::minLayerId + 1; layer0 <= GEMDetId::maxLayerId0; ++layer0) {
+      vector<int> rolls0, rings0, chambers0;
+      for (int ring = GEMDetId::minRingId; ring <= GEMDetId::maxRingId; ++ring)
+        rings0.push_back(ring);
+      for (int chamber = GEMDetId::minChamberId + 1; chamber <= GEMDetId::maxChamberId / 2; chamber++)
+        chambers0.push_back(chamber);
+      for (int roll = GEMDetId::minRollId + 1; roll <= GEMDetId::maxRollId - 6; ++roll)
```

Comment:
End the loop at `EMDetId::maxRollId - 6` if `station = GEMDetId::minStationId0`
(and maybe also explain why this "-6" comes out...)